src/containers/apps/AppsTable.tsx

```diff
@@ -328,7 +328,22 @@ class AppsTable extends Component<
         return (
             <Card
                 extra={
-                    <div>
+                    <div
+                        style={{
+                          display: "flex",
+                          alignItems: "center",
+                          gap: "0.5em"
+                        }}
+                    >
+                        <Button
+                          onClick={() => document.getElementById('create-new-app')?.toggleAttribute('open')}
```

Comment:
i tried to implement the minimalistic change here and relied on basic html5 feature. Since there is no state-tracking possibility of the "open" state of the <dialog> element, the only other approach would be to use useRef. 

Since Apps.tsx and AppsTable.tsx are separated i don't have the exact overview of your hierarchy, so i found that this is the simplest approach. Also, it is just frontend only.

But feel free to adjust it if you are not happy with my approach here.